test/strategies.js

```diff
@@ -4454,6 +4454,225 @@ const createMorphoBlueFLDebtRepayStrategy = () => {
     repayStrategy.addAction(morphoBlueRatioCheckAction);
     return repayStrategy.encodeForDsProxyCall();
 };
+
+const createAaveV3OpenOrderFromCollStrategy = () => {
+    const aaveV3OpenOrderFromCollStrategy = new dfs.Strategy('AaveV3OpenOrderFromCollStrategy');
+
+    aaveV3OpenOrderFromCollStrategy.addSubSlot('&collAsset', 'address');
+    aaveV3OpenOrderFromCollStrategy.addSubSlot('&collAssetId', 'uint16');
+    aaveV3OpenOrderFromCollStrategy.addSubSlot('&debtAsset', 'address');
+    aaveV3OpenOrderFromCollStrategy.addSubSlot('&debtAssetId', 'uint16');
+    aaveV3OpenOrderFromCollStrategy.addSubSlot('&marketAddr', 'address');
+    aaveV3OpenOrderFromCollStrategy.addSubSlot('&targetRatio', 'uint256');
+    aaveV3OpenOrderFromCollStrategy.addSubSlot('&useOnBehalf', 'bool');
+    aaveV3OpenOrderFromCollStrategy.addSubSlot('&rateMode', 'uint8');
+
+    const trigger = new dfs.triggers.ChainLinkPriceTrigger(nullAddress, '0', '0');
```

Comment:
We should probably use AaveV3QuotePriceTrigger in both places instead of checking chainlink price